packages/content-conformance/src/rules/top-of-file-h1.js

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+/** @type {import('../types.js').ConformanceRuleBase} */
+export default {
+  type: 'content',
+  id: 'top-of-file-h1',
+  description: 'Each content file must contain an H1 the top.',
+  executor: {
+    async contentFile(file, context) {
+      file.visit(['root'], (node) => {
+        const firstNode = node.children[0]
+        const reportMessage = `- ${file.path} \nThe file listed above is missing a level 1 heading at the top of the file. Please add a level 1 heading to the top of the file and try again.`
```

Comment:
We don't need the file name in the message here as when we generate the report the message should be in a section headlined by the file name

Reply:
Ah sorry @nayres 🤦 , @BRKalow is right. The filenames are already present. 

https://github.com/hashicorp/web-platform-packages/blob/cf7a69b6db8d607a9e1b73464f191828cbb61c64/packages/content-conformance/src/__tests__/engine.test.ts#L94-L108